src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown

```diff
@@ -25,6 +25,12 @@ The Internet of Things is on everyone's lips these days, and
 it makes tinkerers and programmers like me very excited. Nothing
 is cooler than bringing your own inventions to life and
 being able to talk to them!
+
+But IoT devices that install apps that you rarely use on your phone or PC
+can be annoying. We can take advantage of the Physical Web project, progressive
+apps, and Bluetooth&#42; Low Energy using using Web Bluetooth an Node.js on
+the Intel&#42; Edison board to make IoT devices more intuitive and less
+intrusive.
 </p>
```

Comment:
The second sentence is a bit of a mouth full. Can you spit it, or at least take another shot?

Also on line 31 an >> and